pkg/api/types.go

```diff
@@ -1817,6 +1817,7 @@ const (
 	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
 	ResourceStorage ResourceName = "storage"
 	// Number of Pods that may be running on this Node: see ResourcePods
```

Comment:
This comments seema to be unrelated to the field below. Is github rendering it incorrectly?